### ISSUE TITLE
Making the marker flexible to change coordinates inside a square

### ISSUE THREAD

**Issue by genericlearner** [rima1881/Flat-Earthers-Frontend#16]
We can't change the marker/lat lng coordinates if we are inside a square/squares.

**Comment by rima1881** [comment]
yeah i noticed that. I am not sure if i wanna fix it for now. I an square is selected it doesn't matter where on it is your target worse case they can cancel and reselect. But I agree it is annoying so if we have enough time I go for it

**Comment by genericlearner** [comment]
I was also thinking that, but it is terrible UE if the user cannot change a location. A single square is way too big. However, if our main objective is just the images(it is the main objective). Then we can sideline ts. We got a search bar anyways
